src/main/java/org/testng/reporters/XMLReporter.java

```diff
@@ -26,6 +26,8 @@
 public class XMLReporter implements IReporter {
 
   public static final String FILE_NAME = "testng-results.xml";
+  private static final String JVM_ARG = "xml.report.name";
```

Comment:
There's no clear standard for this but I think we should prefix system properties with `testng`, e.g. `testngXmlReportName` or something like that.